configuration/arm64/initialization.sh

```diff
@@ -1,4 +1,31 @@
 #!/bin/bash
+# update and upgrade commands to update linux OS
+sudo apt-get update -y && sudo apt-get upgrade -y
+sudo apt update -y && sudo apt upgrade -y
+
+#installing necessary and useful apps
+sudo apt-get install chromium-browser -y      #Chrome required for CARMA platform/V2X Hub UI(?)
+sudo apt install curl -y                      #Curl for downloading files over internet
+
+#install docker
+curl -L https://raw.githubusercontent.com/usdot-fhwa-stol/carma-platform/develop/engineering_tools/install-docker.sh | bash 
+
+#make passwords for mysql
+mkdir secrets && cd secrets
+sleep 1
+
+#creates password files where user inputs password
+read -p "enter password for the mysql_root_password: " sql_root_pass
+echo "$sql_root_pass" > sql_root_pass.txt
+
+read -p "enter password for mysql_password: " sql_pass
+echo "$sql_pass" > sql_pass.txt
+
+#remove endline characters from password files
+tr -d '\n' <sql_root_pass.txt> mysql_root_password.txt && tr -d '\n' <sql_pass.txt> mysql_password.txt
+rm sql_root_pass.txt && rm sql_pass.txt
+
+#ARM initialization
 sudo apt-get -y remove docker docker-engine docker.io containerd runc
 sudo apt-get update
```

Comment:
We can probably remove all redundant update calls unless we are adding a apt repository somewhere. We only really new one.